app/controllers/submissions_controller.rb

```diff
@@ -5,7 +5,9 @@ class SubmissionsController < ApplicationController
   before_action -> { authorize_user('challenge_manager') }
   before_action :set_submission, only: [:show, :update]
 
-  def show; end
+  def show
+    @available_evaluators = @submission.phase.evaluators - @submission.evaluators
```

Comment:
this list needs a bit of tweaking to account for different statuses. available evaluators should include evaluators that were assigned and then unassigned. when you add specs we should test all these scenarios where the evaluators are in different states of assigned/unassigned/recused/recused_unassigned and make sure it's working for each case.

Reply:
See this [Figma comment thread](https://www.figma.com/design/VfdBFG8ShVnOStdSNYy9fR?node-id=2992-72444#1072715283).. if the evaluator was unassigned the button CTA should read "Reassign to Submission".